btap/src/predict.py

```diff
@@ -45,6 +45,10 @@
 
 import config as acm
 import plot as pl
+import logging
+
+logging.basicConfig(filename='../output/log/predict.log', level=logging.DEBUG)
```

Comment:
Log data is going to end up in a file, which will be lost when the component is done being evaluated in kubeflow.

```suggestion
logging.basicConfig(level=logging.DEBUG)
```